react/game/src/app/components/Navbar/components/Elements/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { NavLink } from 'react-router-dom';
+
+import styles from '../../styles.scss';
+import LIST from '../../../../../constants/routes';
+import Logout from '../../../Logout';
+
+class Element extends Component {
+  menuList = obj => {
+    const mapped = [];
+    for (const item of Object.values(obj)) { // eslint-disable-line
+      if (item.name !== 'login') {
+        mapped.push(
+          <NavLink
+            exact
+            key={mapped.length}
+            className={styles.navbarMenuLink}
+            activeClassName={styles.navbarMenuLinkActive}
+            to={item.path}
+          >
+            {item.name}
+          </NavLink>
+        );
+      }
+    }
+    return mapped;
+  };
+
+  render() {
+    return (
+      <Fragment>
+        {this.menuList(LIST)}
```

Comment:
Mmmm in each render u are creating a new `mapped` array and running a for for fill this array.  Thats a problem for the performance. U see? Also, this way generates re-renders. I'd use a simple `map` here. U can do this
```js
{LIST.map(this.renderList)}